pkg/yurtctl/cmd/clusterinfo/clusterinfo.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterinfo
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	ct "github.com/daviddengcn/go-colortext"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+
+	"github.com/alibaba/openyurt/pkg/projectinfo"
+	kubeutil "github.com/alibaba/openyurt/pkg/yurtctl/util/kubernetes"
+
+)
+
+
+
+// ClusterInfoOptions has the information that required by cluster-info operation
+type ClusterInfoOptions struct {
+	clientSet             *kubernetes.Clientset
+	CloudNodes            []string
+	EdgeNodes             []string
+}
```

Comment:
how about add another slice to place nodes that neither cloud node nor edge node?

Reply:
OK, I will add it

Reply:
as shown below

![image](https://user-images.githubusercontent.com/19708079/106085444-2a07bb00-615b-11eb-9b7f-56ad1e48b926.png)